lib/IRGen/IRGenModule.h

```diff
@@ -1020,10 +1020,10 @@ class IRGenModule {
                         bool setIsNoInline = false);
 
   llvm::Constant *getOrCreateRetainFunction(const TypeInfo &objectTI, SILType t,
-                                            llvm::Type *llvmType);
+                              llvm::Type *llvmType, Atomicity atomicity);
```

Comment:
@kubamracek in the future can you git-clang-format code? I am pretty sure this didn't get run through git-clang-format.